homework2/fastq_filtrator.py

```diff
@@ -0,0 +1,75 @@
+"""GC-check function"""
+def GC_count_fun(f, gc_bounds = (0, 100)):
+  cnt=0
+  #Counting the amount of cytosine and guanine in a string
+  for nucl in f:
+    if (nucl == 'C') or (nucl == 'G'):
+      cnt += 1
+  #Checking and defining boundaries
+  if type(gc_bounds) == tuple:
+    right_margin = gc_bounds[1]
+    left_margin = gc_bounds[0]
+  else:
+    right_margin = gc_bounds
+    left_margin = 0
+
+  if right_margin >= ((cnt/len(f))*100) >= left_margin:
+     return 1
+  else:
+    return 0
+
+"""Read quality check function"""
+def quality_fun(f, quality_threshold = 0):
+  phred_value=0
+  for symbol in f:
+    phred_value += (ord(symbol) - 33)
+  if quality_threshold <= (phred_value/len(f)):
+    return 1
+  else:
+    return 0
```

Comment:
По PEP8 требуется количество пробелов в оступе кратное 4
```suggestion
    phred_value=0
    for symbol in f:
        phred_value += (ord(symbol) - 33)
    return quality_threshold <= (phred_value /len(f))
```